src/_data/platforms.yml

```diff
@@ -1,50 +1,50 @@
 - platform: 'Android SDK'
   chipsets: 'x64, Arm32, Arm64'
   supported: '21 to 34'
-  besteffort: 'N/A'
+  ci-tested: '21 to 34'
   unsupported: '20 and earlier'
 - platform: 'iOS'
   chipsets: 'Arm64'
-  supported: '17'
-  besteffort: '12 to 16, 18'
+  supported: '12 to 18'
+  ci-tested: '16, 17'
   unsupported: '11 and earlier'
 - platform: 'macOS'
   chipsets: 'x64, Arm64'
-  supported: '<abbr title="Ventura">13</abbr>, <abbr title="Sonoma">14</abbr>'
-  besteffort: '<abbr title="Mojave">10.14</abbr> to <abbr title="Monterey">12</abbr>, <abbr title="Sequoia">15</abbr>'
-  unsupported: '<abbr title="High Sierra">10.13</abbr> and earlier'
+  supported: '10.14 to 10.15'
```

Comment:
Mojave (10.14) to Sequoia (15)

Reply:
Yeah I would be tempted to phrase these consistently with Apple's phrasing, as @tvolkert states.

Other than adding a set of macOS 14 bots as mentioned below. We haven't changed much in terms of our CI config, nor have we dropped support for older versions, so in terms of the way the [Supported deployment platforms](https://docs.flutter.dev/reference/supported-platforms) page is laid out today, I'd say:

* Supported: Ventura (13), Sonoma (14)
* Best efforts versions: Mojave (10.14) to Monterey (12) and Sequoia (15).
* Unsupported: High Sierra (10.13) and earlier.